### PR TITLE
Add Slovakian translation

### PR DESCRIPTION
**Pull request by ricco386**
Our team of volunteers has translated the tutorial some times ago. I have fixed what crowdin broke, but not sure with the gitbook stuff...

### CODE REVIEW

File: sk/css/README.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+
+Bootstrap je jedným z najpopulárnejších HTML a CSS frameworkov pre vývoj krásnych webových stránok: http://getbootstrap.com/
+
+Bol napísaný programátormi, ktorí pracovali pre Twitter a ďalej ho vyvíjajú dobrovoľníci z celého sveta.
+
+## Inštalácia Bootstrapu
+
+Na inštaláciu Bootstrapu musíš pridať do hlavičky `<head>` vo svojom `.html` súbore (`blog/templates/blog/post_list.html`) toto:
+
+    html
```
Comment (tricoder42) [inline sk/css/README.md]: This line should start with triple apostrophes:
```
```html
<code block here>
```

and the same (triple) apostrophes should be below code block too (to close highlighted area).

File: sk/css/README.md [hunk 2]
```diff
@@ -0,0 +1,293 @@
...
+
+ [2]: http://www.codecademy.com/tracks/web
+
+Ale urobme aspoň niečo. Čo keby sme zmenili farbu hlavičky? Počítače používajú špeciálne kódy, aby rozumeli farbám. Začínajú `#`, potom nasleduje 6 písmen (A-F) a číslic (0-9). Kódy farieb nájdeš napríklad tu: http://www.colorpicker.com/. Môžeš tiež použiť [preddefinované farby][3] ako napríklad `red` (červená) alebo `green` (zelená).
+
+ [3]: http://www.w3schools.com/cssref/css_colornames.asp
+
+V súbore `blog/static/css/blog.css` pridaj nasledujúci kód:
+
+    css
```
Comment (tricoder42) [inline sk/css/README.md]: Same here, missing apostrophes:

```
```css
```

File: sk/python_installation/instructions.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+> Táto časť je založená na príručke Geek Girls Carrots (http://django.carrots.pl/)
+
+Django je napísané v Pythone. Python potrebujeme na všetko, čo chceme robiť v Djangu. Začnime teda s inštaláciou! Chceme, aby si si nainštalovala Python 3.4, takže ak máš staršiu verziu, budeš ju musieť upgradovať.
+
+### Windows
+
+Python pre Windows si môžeš stiahnuť z webstránky https://www.python.org/downloads/release/python-343/. Po stiahnutí súboru ***.msi**, ho spusti (dvojklikom) a postupuj podľa inštrukcií. Je dôležité zapamätať si cestu (adresár), kde je nainštalovaný Python. Ešte to budeme potrebovať!
```
Comment (tricoder42) [inline sk/python_installation/instructions.md]: Missing one asterisk after `.msi` which makes the rest of the page in **bold**.

**Comment by tricoder42** [comment]
There's missing code block syntax in these files (basically in every file which has at least one code block with syntax highlighting):

- [ ] sk/css/README.md
- [ ] sk/deploy/README.md
- [ ] sk/django_admin/README.md
- [ ] sk/django_forms/README.md
- [ ] sk/django_models/README.md
- [ ] sk/django_start_project/README.md
- [ ] sk/django_templates/README.md
- [ ] sk/django_urls/README.md
- [ ] sk/django_views/README.md
- [ ] sk/dynamic_data_in_templates/README.md
- [ ] sk/extend_your_application/README.md
- [ ] sk/html/README.md
- [ ] sk/python_introduction/README.md
- [ ] sk/template_extending/README.md

Shell and directory tries doesn't require syntax highlighting, but everything else do.

**Comment by patjouk** [comment]
Hi! @ricco386 when you have time, can you fix the missing code block syntax? It would be really cool to merge this translation. :sparkles: 
Thanks!

**Comment by ricco386** [comment]
Hey @patjouk @tricoder42 I have fixed missing apostrophes so it should be ok now. Could you please review the merge request?

**Comment by aniav** [comment]
@ricco386 could you please rebase with the current version of tutorial? Thank you! :))

**Comment by ricco386** [comment]
@aniav I have merged master branch into sk branch so you should be able to merge to master without conflicts...

File: sk/deploy/README.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+
+```
+    $ git add -A .
+    $ git commit -m "My Django Girls app, first commit"
+     [...]
+     13 files changed, 200 insertions(+)
+     create mode 100644 .gitignore
+     [...]
+     create mode 100644 mysite/wsgi.py
+
```
Comment (aniav) [inline sk/deploy/README.md]: The closing fencing is missing here.

**Comment by aniav** [comment]
That is now merged! Thank you all for work on this translation ❤️

**Pull request merged**